src/index.ts

```diff
@@ -215,11 +215,17 @@ class WalletLedger {
     fromAccount: Account,
     dest: string,
     amount: BigNumber,
-    fee: number,
+    feePerByte: number,
     utxoPickingStrategy: IPickingStrategy
   ) {
     const changeAddress = await fromAccount.xpub.getNewAddress(1, 1);
-    const txinfos = await fromAccount.xpub.buildTx(dest, amount, fee, changeAddress.address, utxoPickingStrategy);
+    const txinfos = await fromAccount.xpub.buildTx(
+      dest,
+      amount,
+      feePerByte,
```

Comment:
just for a reminder, we need to expose a method to get the result from https://explorers.api.live.ledger.com/blockchain/v3/{coin}/fees later